c7n/resources/account.py

```diff
@@ -367,6 +367,51 @@ def process(self, resources, event=None):
         return []
 
 
+@filters.register('access-analyzer')
+class AccessAnalyzer(Filter):
+    """Check for access analyzers in an account
+
+    :example:
+
+    .. code-block:: yaml
+
+            policies:
+              - name: account-access-analyzer
+                resource: account
+                region: us-east-1
+                filters:
+                  - type: access-analyzer
+                    status: active
+                    trust: organization
+    """
+
+    schema = type_schema('access-analyzer',
+                        status={'oneOf': [{'enum': ['active', 'creating', 'disabled', 'failed']}]},
```

Comment:
IMO, status should be a required field here

Reply:
its already defaulting to active which is pretty sane and captures what most people want.

Reply:
the trust one is a little more interesting, as it should probably default to not being checked.